src/main/java/com/jcraft/jsch/KeyExchange.java

```diff
@@ -54,7 +58,7 @@ public abstract class KeyExchange {
   static String enc_c2s = "blowfish-cbc";
   static String enc_s2c = "blowfish-cbc";
   static String mac_c2s = "hmac-md5"; // hmac-md5,hmac-sha1,hmac-ripemd160,
-                                      // hmac-sha1-96,hmac-md5-96
+  // hmac-sha1-96,hmac-md5-96
```

Comment:
Can we restore the leading space here?
Otherwise it make it appear that this comment incorrectly applies the next variable.